packages/app-store/stripepayment/zod.ts

```diff
@@ -18,6 +18,7 @@ export const appDataSchema = eventTypeAppCardZod.merge(
     currency: z.string(),
     paymentOption: paymentOptionEnum.optional(),
     enabled: z.boolean().optional(),
+    credentialId: z.number().optional(),
```

Comment:
https://github.com/calcom/cal.com/blob/1610d083428ddda277ffbfdbf4f11fc4537eba26/packages/app-store/eventTypeAppCardZod.ts#L6

eventTypeAppCardZod already has `credentialId`, so we shouldn't be needing it.